docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 web:
-  image: grahamc/jekyll:latest
+  build: .
   ports:
     - "127.0.0.1:4000:4000"
```

Comment:
This is a docker for dev, not for prod though. I thought that prod docker would be in another 'infra' repo. That might be a good discussion to have this week :-)